src/Test/LicTest.java

```diff
@@ -63,6 +63,51 @@ void licFive() {
 
     @Test
     void licSix() {
+        //all points same distance away
+        Point2D.Double[] points = {
+                new Point2D.Double(0, 0 ),
+                new Point2D.Double(0,0),
+                new Point2D.Double(0, 0),
+                new Point2D.Double(0,0),
+                new Point2D.Double(0, 0)
+        };
+        params.setPoints(points);
+        assertFalse(Lic.licSix(params));
+
+        //First and last point the same, one point further than dist away from the coincident point
+        Point2D.Double[] points2 = {
+                new Point2D.Double(0, 0 ),
+                new Point2D.Double(0,0),
+                new Point2D.Double(0, params.getDist()*2),
+                new Point2D.Double(0,0),
```

Comment:
puts the outlier point at the end of the inner for-loop of licSix. Also reduced its distance from the coincident point to make it more like an edge case.
```suggestion
                new Point2D.Double(0,params.getDist() + 0.1),
```